test/distribution/test_distribution_output_shapes.py

```diff
@@ -141,6 +142,13 @@
             (3, 4),
             (5,),
         ),
+        (
+            PoissonOutput(),
+            mx.nd.random.poisson(shape=(3, 4, 5, 6)),
```

Comment:
This was my bad, apologies: this tensor is supposed to be the output of the network, which will be automatically mapped to the parameter domain (positive reals) by the `PoissonOutput` class. I mistakenly though this was supposed to be data in the support of the distribution.

```suggestion
            mx.nd.random.normal(shape=(3, 4, 5, 6)),
```